src/components/Accounts/Menu.js

```diff
@@ -76,31 +72,25 @@ const Menu = (props) => {
         <Col className={css.firstMenuItems}>
           <div id="outstanding-balance">
             <FormattedMessage
-              id="ui-users.accounts.outstanding"
-              values={{
-                amount: outstanding
-              }}
+              id="ui-users.accounts.outstanding.page"
+              values={{ amount: outstanding }}
             />
-            &nbsp; (
+            &nbsp;|&nbsp;
```

Comment:
Could you please use css for separator (&nbsp;|&nbsp;)? Example below:
<span className={css.separator}>|</span> .separator {  padding: 0 3px; }

Reply:
I don't really understand in which way adding of extra `span` tag and styles would be more appropriate instead of html entity (whitespace).